Source/WebCore/Modules/mediastream/gstreamer/GStreamerRtpReceiverBackend.cpp

```diff
@@ -76,7 +76,7 @@ RTCRtpParameters GStreamerRtpReceiverBackend::getParameters()
         auto media = gstStructureGetString(structure, "media"_s);
         auto encodingName = gstStructureGetString(structure, "encoding-name"_s);
         if (!media.isEmpty() && !encodingName.isEmpty())
-            codec.mimeType = makeString(media.toString(), '/', encodingName.toString().convertToASCIILowercase());
+            codec.mimeType = makeString(toString(media), '/', convertToASCIILowercase(encodingName.span()));
```

Comment:
This can be done without two temporary `String` objects.
```suggestion
            codec.mimeType = makeString(media.span(), '/', convertToASCIILowercase(encodingName.span()));
```
But also, some day the second one can also be done without a temporary `String` object if we make an `ASCIICaseConverter` that works on UTF-8, like the one I already made for `StringView`.